### ISSUE TITLE
Boundaries self-moving bug

### ISSUE THREAD

**Issue by PavelLysenok** [lgsvl/simulator#1280]
Hi again!

I mentioned a very strange bug that popped up suddenly. When I click on a boundary line and turn on the Toggle Handles, all the waypoints begin to move along two axes without any manipulations with them. It only appears when I select the boundaries - all the lanes, for example, stay at the same position. It's rather difficult to explain so I attach the screen capture video file:

https://drive.google.com/file/d/1q5EpzN_j20NAcMzoazhzjjUE33uo9anU/view?usp=sharing

Not a single key is pressed (I checked in the TextEditor twice). Restarting both Unity and the computer does not help.

**Comment by EricBoiseLGSVL** [comment]
Yes, this is a bug with floating point precision in the editor.  We are looking into a fix for this and hope to have it fixed soon.
